test/tools/runner/flaky.ts

```diff
@@ -32,5 +32,6 @@ export const flakyTests = [
   'CSOT spec tests timeoutMS behaves correctly for GridFS download operations timeoutMS applied to entire download, not individual parts',
   'Retryable Writes (unified) retryable writes handshake failures collection.updateOne succeeds after retryable handshake network error',
   'Retryable Reads (unified) retryable reads handshake failures collection.aggregate succeeds after retryable handshake network error',
-  'CSOT spec tests legacy timeouts behave correctly for retryable operations operation fails after two consecutive socket timeouts - aggregate on collection'
+  'CSOT spec tests legacy timeouts behave correctly for retryable operations operation fails after two consecutive socket timeouts - aggregate on collection',
+  'Server Discovery and Monitoring Prose Tests Connection Pool Management ensure monitors properly create and unpause connection pools when they discover servers'
```

Comment:
Note test wasn't already skipped, it's running: https://github.com/mongodb/node-mongodb-native/blob/main/test/integration/server-discovery-and-monitoring/server_discovery_and_monitoring.prose.test.ts#L149